src/components/settings/Settings.h

```diff
@@ -124,10 +124,19 @@ namespace Pinetime {
       
       uint32_t GetStepsGoal() const { return settings.stepsGoal; };
 
+      void SetHeartSensorGain( uint32_t gain ) {
+        if ( gain != settings.heartSensorGain ) {
```

Comment:
Please format the files using clang-format

Reply:
Newer used clang-format, and I saw at first too many unrelated changes and decided to skip it... Hopefully now its okay? Later I can squash commits

Reply:
Looks better now, yeah. There's a clang-format configuration file in the repository, changes made using it are fine. Ideally there's only new lines to format, but sometimes inconsistent lines have slipped in.